### PR TITLE
Update 0x06f-Testing-Local-Authentication.md

### PR DESCRIPTION
**Pull request by colman-mbuya**
Fixed a couple of typos and grammar errors. Added Needle as an alternative to bypass Touch ID

### CODE REVIEW

File: Document/0x06f-Testing-Local-Authentication.md [hunk 1]
```diff
@@ -49,9 +49,9 @@ context.evaluatePolicy(.deviceOwnerAuthentication, localizedReason: "Please, pas
 
 #####  Using Keychain Services for Local Authentication
 
-The iOS Keychain APIs can (and should) be used to implement local authentication. During this process, the app requests either a secret authentication token or another piece of secret data identifying the user stored by the Keychain. In order to authenticate a remote service, the user must unlock the Keychain using their passphrase or fingerprint to obtain the secret data. 
+The iOS Keychain APIs can (and should) be used to implement local authentication. During this process, the app requests either a secret authentication token or another piece of secret data identifying the user be stored in the Keychain. In order to authenticate to a remote service, the user must unlock the Keychain using their passphrase or fingerprint to obtain the secret data.
```
Comment (Sjord) [inline Document/0x06f-Testing-Local-Authentication.md]: I think this sentence can become a little bit simpler: "the app stores ... in the Keychain", or "the app requests the Keychain to store ...".

File: Document/0x06f-Testing-Local-Authentication.md [hunk 2]
```diff
@@ -206,6 +206,8 @@ On a jailbroken device tools like [Swizzler2](https://github.com/vtky/Swizzler2
 - When the TouchID prompt shows click "cancel"
 - If the application flow continues without requiring the touchID then the bypass has worked.
 
+If you're using Needle, run the "hooking/frida/script_touch-id-bypass" module and follow the prompts. This will spawn the application and instrument the `evaluatePolicy` function. When prompted to authenticate via Touch ID, tap cancel. If the application flow continues, then you will have successfully bypassed Touch ID. A similar module (hooking/cycript/cycript_touchid) that uses cycript instead of frida is also available in Needle.
```
Comment (Sjord) [inline Document/0x06f-Testing-Local-Authentication.md]: > then you will have successfully bypassed Touch ID

I think "will" can be removed here, and maybe "have" too: "then you successfully bypassed Touch ID".

File: Document/0x06f-Testing-Local-Authentication.md (hunk 1 above)
Comment (colman-mbuya) [reply Document/0x06f-Testing-Local-Authentication.md]: Agreed. Have changed the sentence

File: Document/0x06f-Testing-Local-Authentication.md (hunk 2 above)
Comment (colman-mbuya) [reply Document/0x06f-Testing-Local-Authentication.md]: I think it sounds better with "have". I have removed the word "will"

**Pull request merged**